pkg/framework/aries/context.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import "github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+
+// Context provides handles to various configs to be used by the clients
+type Context struct {
```

Comment:
FYI: in Fabric SDK Go, we had the context implementation in its own [package](https://github.com/hyperledger/fabric-sdk-go/tree/master/pkg/context).

Thoughts?

CC: @sudeshrshetty @Baha-sk @fqutishat @bstasyszyn

Reply:
Another example is the AWS v2 SDK.

The config object has some similarity: https://github.com/aws/aws-sdk-go-v2/blob/master/aws/config.go

(and is kept in the same package).

Reply:
Note: this Context is an Application Context as opposed to the stdlib context.Context, which is a Request Context / Net Context (for cancellation).

Reply:
@troyronda  @sudeshrshetty [updated code](https://github.com/hyperledger/aries-framework-go/compare/cf2367d04a6dbd7e0196b81fcf24ec5a1df59ae9..10f3e059cd136f4f1f0dd4d06539cd51a8b4f3e4) to follow fab-sdk structure.

Reply:
Created #115.